arborist/aws_client.go

```diff
@@ -0,0 +1,142 @@
+package arborist
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+// S3Credentials contains AWS credentials
+type S3Credentials struct {
+	region             string
+	awsAccessKeyID     string
+	awsSecretAccessKey string
+}
+
+type AwsClient struct {
+	credentials S3Credentials
+	session     *session.Session
+	bucket      string
+}
+
+func (client *AwsClient) SetS3Credentials(cred S3Credentials) {
```

Comment:
this function not used

Reply:
Fixed in bc25eb4ad8ccd44d86b7a24694e3989552e789c7